charts/csi-powerstore/values.yaml

```diff
@@ -47,13 +47,13 @@ images:
 
   # CSM sidecars
   replication:
-    image: quay.io/dell/container-storage-modules/dell-csi-replicator:v1.11.0
+    image: quay.io/dell/container-storage-modules/dell-csi-replicator:v1.12.0
   vgsnapshotter:
-    image: quay.io/dell/container-storage-modules/csi-volumegroup-snapshotter:v1.8.1
+    image: quay.io/dell/container-storage-modules/csi-volumegroup-snapshotter:v1.9.0
```

Comment:
VGS will be deprecated in CSM 1.14. Do we release it this time? Please check with POs.

Reply:
@santhoshatdell: Nice catch, it is reverted now. Please re-review.